wordpress-mobile/WordPress-Android
# Scan History OnClick action

Parent issue #13326

Implements on list item click action on Scan History screen.

To test:
1. Make sure Scan feature flag is enabled
2. Open Scan
3. Click on History action in the Toolbar
4. Wait until the items get fetched (it might take 30+seconds)
5. Click on one of the items and notice Threat Detail screen is shown

Note: Ignore all styling.

PR submission checklist:
